### PR TITLE
[WIP] Baremetal: CoreOS Live image based installs

### PR DESCRIPTION
**Pull request by kirankt**
This PR adds support for Baremetal IPI CoreOS Live image-based installation.

### CODE REVIEW

File: data/data/bootstrap/baremetal/files/usr/local/bin/startironic.sh.template [hunk 1]
```diff
@@ -153,11 +142,36 @@ $IPTABLES -t raw -A DHCP_IRONIC -j DROP
...
+     -v $IRONIC_SHARED_VOLUME:/shared:z ${IRONIC_IMAGE}
+
+# Run httpd here to populate the IRONIC_KERNEL_PARAMS
+# Note: the IRONIC_HTTPD while set to false, will configure the KERNEL params
+# but not start the httpd server.
+podman run -d --net host --privileged --name httpd \
+     --restart on-failure \
+     --env IRONIC_RAMDISK_SSH_KEY="$IRONIC_RAMDISK_SSH_KEY" \
+     --env PROVISIONING_INTERFACE=$PROVISIONING_NIC \
+     --env IRONIC_HTTPD=true \
```
Comment (elfosardo) [inline data/data/bootstrap/baremetal/files/usr/local/bin/startironic.sh.template:164]: this should be needed only if the entrypoint is ironic-api
/bin/runhttpd will always start the httpd server

Comment (kirankt) [reply data/data/bootstrap/baremetal/files/usr/local/bin/startironic.sh.template:164]: +1. Sorry I got things mixed up.

**Pull request closed without merging**